### PR TITLE
Fix: Revert #26931 change due to new bug it introduced

### PR DESCRIPTION
**Pull request by micahgodbolt**
fixes https://github.com/microsoft/fluentui/issues/27256

#26931 made a change where local state in autofill would not update if value was being controlled, as this caused a bug where cursor would jump to the end. https://github.com/microsoft/fluentui/issues/18499

Unfortunately, IME support relied on local state always being updated, so this broke IME support for Combobox in freeform mode.

As both bugs are affecting the same customer, they would rather revert and take on the smaller previous bug.

To address the original issue we need to determine how we can return the original behavior while not breaking IME (it looks like getDerivedStateFromProps is getting called multiple times during IME and sometimes the updateInputValue is an empty string, leaving the value empty in the end)

After this change is reverted we'll need to cherrypick it back to 8.106.1

### CODE REVIEW

File: change/@fluentui-react-c04deae4-a13e-45a0-8a98-b7c3f4ca4a5b.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "revert pr #26931",
```
Comment (behowell) [inline change/@fluentui-react-c04deae4-a13e-45a0-8a98-b7c3f4ca4a5b.json:3]:
```suggestion
  "comment": "Fix: Fix issue with ComboBox not accepting IME input (revert PR #26931)",
```

File: change/@fluentui-react-c04deae4-a13e-45a0-8a98-b7c3f4ca4a5b.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```
Comment (behowell) [inline change/@fluentui-react-c04deae4-a13e-45a0-8a98-b7c3f4ca4a5b.json:2]: The change was was 'minor' in #26931 -- should this also be 'minor'?
https://github.com/microsoft/fluentui/pull/26931/files#diff-57e25edb4a3e974d4a4f48a6814a05ca73763c9b6e057734fdaf5479b8d7d33e

**Review by behowell — APPROVED** [review]

**Comment by behowell** [comment]
Do you also want to mark this as fixing #27256? Or would you rather keep that issue active for when you're reapplying the fix for the original issue?

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/df9564ee42c12b0dfab646f9e4dd4038c636d3ee'>df9564ee42c12b0dfab646f9e4dd4038c636d3ee</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test-react-components/Avatar-mount_base.html">887</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test-react-components/Avatar-mount.html">893</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test-react-components/Button-mount_base.html">592</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test-react-components/Button-mount.html">592</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Field</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test-react-components/Field-mount_base.html">1568</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test-react-components/Field-mount.html">1552</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test-react-components/FluentProvider-mount_base.html">1111</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test-react-components/FluentProvider-mount.html">1098</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test-react-components/FluentProviderWithTheme-mount_base.html">293</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test-react-components/FluentProviderWithTheme-mount.html">292</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">268</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">279</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">299</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">286</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>InfoButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test-react-components/InfoButton-mount_base.html">202</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test-react-components/InfoButton-mount.html">200</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test-react-components/MakeStyles-mount_base.html">1383</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test-react-components/MakeStyles-mount.html">1367</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test-react-components/Persona-mount_base.html">2142</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test-react-components/Persona-mount.html">2063</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test-react-components/SpinButton-mount_base.html">1840</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test-react-components/SpinButton-mount.html">1814</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

File: change/@fluentui-react-c04deae4-a13e-45a0-8a98-b7c3f4ca4a5b.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (fabricteam) [inline change/@fluentui-react-c04deae4-a13e-45a0-8a98-b7c3f4ca4a5b.json:1]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the 4 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27323&commitId=2e76225729c55b853bf09053b3265be6b362b950&env=prod)

✅ There was 0 screenshots added, 4 screenshots removed, 1043 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pivot - Overflow.Root.Narrow - Last tab selected.chromium.png|0| Removed|
|Pivot - Overflow.Root.Narrow - Overflow menu.chromium.png|0| Removed|
|Pivot - Overflow.Tabs - RTL.Narrow - Last tab selected.chromium.png|0| Removed|
|Pivot - Overflow.Tabs - RTL.Narrow - Overflow menu.chromium.png|0| Removed|

</details>


<div id="vrtCommentfluentuiv8"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/BaseButton-mount_base.html">946</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/BaseButton-mount.html">948</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/Breadcrumb-mount_base.html">2129</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/Breadcrumb-mount.html">2148</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/Checkbox-mount_base.html">2158</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/Checkbox-mount.html">2135</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/CheckboxBase-mount_base.html">1925</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/CheckboxBase-mount.html">1948</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/ChoiceGroup-mount_base.html">3622</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/ChoiceGroup-mount.html">3629</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/ComboBox-mount_base.html">933</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/ComboBox-mount.html">939</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/CommandBar-mount_base.html">7838</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/CommandBar-mount.html">7826</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/ContextualMenu-mount_base.html">12180</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/ContextualMenu-mount.html">12581</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/DefaultButton-mount_base.html">1089</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/DefaultButton-mount.html">1071</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/DetailsRow-mount_base.html">2811</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/DetailsRow-mount.html">2797</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/DetailsRowFast-mount_base.html">2783</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/DetailsRowFast-mount.html">2834</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/DetailsRowNoStyles-mount_base.html">2589</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/DetailsRowNoStyles-mount.html">2619</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/Dialog-mount_base.html">3091</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/Dialog-mount.html">3133</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/DocumentCardTitle-mount_base.html">425</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/DocumentCardTitle-mount.html">436</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/Dropdown-mount_base.html">2594</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/Dropdown-mount.html">2609</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/FocusTrapZone-mount_base.html">1545</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/FocusTrapZone-mount.html">1562</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/FocusZone-mount_base.html">1546</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/FocusZone-mount.html">1541</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/GroupedList-mount_base.html">45495</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/GroupedList-mount.html">53028</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/GroupedList-virtual-rerender_base.html">22293</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/GroupedList-virtual-rerender.html">22319</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/GroupedList-virtual-rerender-with-unmount_base.html">63080</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/GroupedList-virtual-rerender-with-unmount.html">62655</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/GroupedListV2-mount_base.html">402</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/GroupedListV2-mount.html">410</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/GroupedListV2-virtual-rerender_base.html">388</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/GroupedListV2-virtual-rerender.html">393</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/GroupedListV2-virtual-rerender-with-unmount_base.html">421</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/GroupedListV2-virtual-rerender-with-unmount.html">420</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/IconButton-mount_base.html">1461</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/IconButton-mount.html">1467</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/Label-mount_base.html">569</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/Label-mount.html">570</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/Layer-mount_base.html">3484</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/Layer-mount.html">3502</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/Link-mount_base.html">640</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/Link-mount.html">651</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/MenuButton-mount_base.html">1286</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/MenuButton-mount.html">1304</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/MessageBar-mount_base.html">26253</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/MessageBar-mount.html">26019</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/Nav-mount_base.html">2483</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/Nav-mount.html">2471</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/OverflowSet-mount_base.html">1112</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/OverflowSet-mount.html">1123</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/Panel-mount_base.html">2065</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/Panel-mount.html">2066</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/Persona-mount_base.html">1042</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/Persona-mount.html">1046</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/Pivot-mount_base.html">1229</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/Pivot-mount.html">1219</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/PrimaryButton-mount_base.html">1192</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/PrimaryButton-mount.html">1200</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/Rating-mount_base.html">5761</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/Rating-mount.html">5793</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/SearchBox-mount_base.html">1195</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/SearchBox-mount.html">1210</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/Shimmer-mount_base.html">2427</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/Shimmer-mount.html">2442</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/Slider-mount_base.html">1742</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/Slider-mount.html">1763</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/SpinButton-mount_base.html">3595</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/SpinButton-mount.html">3531</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/Spinner-mount_base.html">641</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/Spinner-mount.html">631</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/SplitButton-mount_base.html">2366</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/SplitButton-mount.html">2336</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/Stack-mount_base.html">673</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/Stack-mount.html">662</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/StackWithIntrinsicChildren-mount_base.html">1200</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/StackWithIntrinsicChildren-mount.html">1215</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/StackWithTextChildren-mount_base.html">3517</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/StackWithTextChildren-mount.html">3524</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/SwatchColorPicker-mount_base.html">7575</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/SwatchColorPicker-mount.html">7618</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/TagPicker-mount_base.html">1860</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/TagPicker-mount.html">1878</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/Text-mount_base.html">616</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/Text-mount.html">626</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/TextField-mount_base.html">1256</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/TextField-mount.html">1249</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/ThemeProvider-mount_base.html">1110</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/ThemeProvider-mount.html">1096</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/ThemeProvider-virtual-rerender_base.html">904</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/ThemeProvider-virtual-rerender.html">905</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/ThemeProvider-virtual-rerender-with-unmount_base.html">1643</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/ThemeProvider-virtual-rerender-with-unmount.html">1630</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/Toggle-mount_base.html">877</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/Toggle-mount.html">899</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>buttonNative</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/buttonNative-mount_base.html">364</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27323/perf-test/buttonNative-mount.html">376</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv8"/>

**Comment by behowell** [comment]
Force merging: the build failures are timeouts in v9 SSR tests, which are unrelated to this change. This is a high priority issue to resolve, so we can't wait for the tests to be fixed.

**Pull request merged**